dev/new-rule/new-rule.js

```diff
@@ -0,0 +1,58 @@
+/* eslint-disable eslint-comments/disable-enable-pair */
+/* eslint-disable node/no-unsupported-features/node-builtins */
+/* eslint-disable prefer-template */
+
+/**
+node.js-CLI-based generation of new rule template files
+Example usage: 
+--------------
+Templates use the rule name `my-new-rule`, so this example
+creates a functioning/immediately testable rule
+# /path/to/ember-template-lint
+$ yarn new my-new-rule
+$ yarn test my-new-rule
+*/
+
+const fs = require('fs');
+const path = require('path');
+
+// CLI Argument: `my-new-rule` (name of new rule to add)
+let newRuleName = process.argv[2];
+
+// Utility Filesystem: template files co-localized with this script
+let pathUtil = __dirname;
+let pathDocTemplate = path.join(pathUtil, 'new-rule-doc-template.md');
+let pathRuleTemplate = path.join(pathUtil, 'new-rule-template.js');
+let pathTestTemplate = path.join(pathUtil, 'new-rule-test-template.js');
+
+// Parent Filesystem: path/to/ember-template-lint
+let pathRoot = process.cwd();
+let pathDocList = path.join(pathRoot, 'docs/', 'rules.md');
+let pathDocNew = path.join(pathRoot, 'docs/rule/', newRuleName + '.md');
```

Comment:
When you use `path.join`, you should omit the slashes, just use: `path.join(pathRoot, 'docs', 'rules', newRuleName + '.md')` for example.